### PR TITLE
Add proxy mapping to get bound monitors

### PR DESCRIPTION
**Pull request by itzg**
# What

Proxy the get bound monitors operation added in https://github.com/racker/salus-telemetry-monitor-management/pull/20

### CODE REVIEW

File: public/src/main/java/com/rackspace/salus/telemetry/api/web/MonitorsController.java [hunk 1]
```diff
@@ -51,15 +50,13 @@ public MonitorsController(UserService userService, ServicesProperties servicesPr
 
   @GetMapping("/monitors")
   public ResponseEntity<?> getAll(ProxyExchange<?> proxy,
-                                                                    @RequestParam(defaultValue = "100") int size,
-                                                                    @RequestParam(defaultValue = "0") int page) {
+                                  @RequestParam MultiValueMap<String,String> queryParams) {
```
Comment (jjbuchan) [inline public/src/main/java/com/rackspace/salus/telemetry/api/web/MonitorsController.java]: What are you planning to do with the size/page defaults?  After your last PR (that didn't include them) I was wondering if we should remove those from the individual repos and specify them in here instead...

I guess the way we're proxying means it makes it easier to have them in the individual services?

Comment (jjbuchan) [reply public/src/main/java/com/rackspace/salus/telemetry/api/web/MonitorsController.java]: Probably only relevant for the size, which defaults to 20.  I imagine the page will always default to 0.

Comment (itzg) [reply public/src/main/java/com/rackspace/salus/telemetry/api/web/MonitorsController.java]: Yes, I would rather let the individual backend REST API declare the defaults for those. When introducing the proxy layer here I didn't like having to re-specify the `Pageable` parameters, but hadn't thought about grabbing the query parameters like the new change does.

**Comment by itzg** [comment]
I forgot this was still open :)

**Pull request merged**